js/src/lib/deeplink.js

```diff
@@ -0,0 +1,30 @@
+export function handleTabs() {
```

Comment:
I did this because I imagine that pages other than the install page could benefit from having the handling of tabs into the history